app/spaces/create/components/constant.tsx

```
@@ -0,0 +1,65 @@
+    AirTrafficControl,
+    IconProps
+} from "@phosphor-icons/react";
+import { Categories as SpaceTypesConstant } from "@/constant";
+
+export const Categories = SpaceTypesConstant.map((i): {
+    value: string;
+    label: string;
+    icon: React.ReactElement<IconProps>;
+    color: string;
+} => {
+    let icon = <UsersThree />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)